tests/runtest.py

```diff
@@ -845,6 +851,11 @@ def parse_argument():
     else:
         flags = arg.flags.split()
 
+    if arg.compiler == 'gccclang':
+        compilers = ['gcc', 'clang']
```

Comment:
We better check if `gcc` and `clang` are installed in the system. If not, we should use only an available compiler.